src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,68 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final int NUMBER_ZERO = 0;
+    private static final String DELIMITER_DEFAULT = ",|:";
+    private static final String DELIMITER_PIPE = "|";
+    private static final String REGEX_CUSTOM_DELIMITER = "//(.)\n(.*)";
+
+    private StringAddCalculator() {
+    }
+
+    public static int splitAndSum(String inputString) {
+        if (isEmpty(inputString)) {
+            return 0;
```

Comment:
```suggestion
            return NUMBER_ZERO;
```
정의해둔 상수를 사용해봐도 좋을 것 같아요 😄

Reply:
감사합니다. 다음 단계에서 반영하겠습니다. 😄